### PR TITLE
feat: file upload and conditional question

### PR DESCRIPTION
**Pull request by himkwtn**

### CODE REVIEW

File: src/file/file.service.ts [hunk 1]
```diff
@@ -69,4 +69,19 @@ export class FileService implements MulterOptionsFactory {
             }
         });
     }
+
+    delete(key: string) {
+        const bucket = this.configService.awsS3BucketName;
+        const params = { Bucket: bucket, Key: key };
+        return this.S3.deleteObject(params).promise();
```
Comment (fivechjr) [inline src/file/file.service.ts]: Do we need to do some try/catch here?

Comment (fivechjr) [reply src/file/file.service.ts]: We might also need to check file existence first. (?)

Comment (himkwtn) [reply src/file/file.service.ts]: Sure, what should I do when there is error?

Comment (himkwtn) [reply src/file/file.service.ts]: If it does not exist, then deleting it should not matter.

**Pull request merged**